semgrep/Makefile

```diff
@@ -1,7 +1,3 @@
-all:
```

Comment:
These commands were only used in `make all` in `ubuntu-release.sh` and `osx-release.sh`.